visgraph/graphcore.py

```diff
@@ -95,6 +97,25 @@ def fromJson(self, graph):
             node2 = (n2, graph['nodes'][n2])
             self.addEdge(node1, node2, eid=eid, eprops=eprops)
 
+    def clone(self, graph):
```

Comment:
seems kinda counter intuitive to have a function on a graph named "clone" that *absorbs* the other graph.  is the intent here to be able to "copy a graph"?  If so, it seems *way* more intuitive to have a clone() method that returns a new graph which is a clone of the one you called it on...